StreamSaver.js

```diff
@@ -17,7 +21,8 @@
 			major: 0, minor: 2, dot: 0
 		}
 	},
-	proxy = 'https://jimmywarting.github.io/StreamSaver.js/mitm.html?version=' +
+	// proxy = 'https://jimmywarting.github.io/StreamSaver.js/mitm.html?version=' +
+	proxy = 'http://localhost:3001/mitm.html?version=' +
 	         streamSaver.version.full
```

Comment:
Nah, just temporary switched off for local development
The service worker has also been turned of.